snuba/schemas.py

```diff
@@ -123,9 +82,44 @@
             },
             'default': [],
         },
+        'conditions': {
+            'type': 'array',
+            'items': {
+                'anyOf': [
+                    {'$ref': '#/definitions/condition'},
+                    {
+                        'type': 'array',
+                        'items': {'$ref': '#/definitions/condition'},
+                    },
+                ],
+            },
+            'default': [],
+        },
         'arrayjoin': {
             '$ref': '#/definitions/column_name',
         },
```

Comment:
Not necessarily related to this change, but I'd like to take a look later and see if we actually use `arrayjoin`. If not, I think we should remove it so that we have more flexibility with implementation decisions when we actually implement joins as part of the query structure.